src/docfx/build/page/BuildPage.cs

```diff
@@ -258,7 +258,20 @@ private static JObject LoadSchemaDocument(ErrorBuilder errors, Context context,
             }
 
             var schema = context.TemplateEngine.GetSchema(mime);
-            var pageModel = (JObject)context.JsonSchemaTransformer.TransformContent(errors, schema, file, validatedObj);
+            var pageModel = (JObject)context.JsonSchemaTransformer.TransformContent(
+                errors.With(e =>
+                {
+                    if (!string.IsNullOrEmpty(e.Name) &&
```

Comment:
is that property name or property json path, I think it's different in metadata validation and yaml validation, should we separate them?

Reply:
OK. Talk with Yufei, got the conclusion which will be changed to PropertyPath in another mini PR.